### ISSUE TITLE
模型输出的值域不在[0,1]

### ISSUE THREAD

**Issue by davidL-zhan** [caiyuanhao1998/Retinexformer#223]
作者您好 ，非常感谢您的代码 我做简单的训练后 模型输出的值域不在[0,1] 大概在【-0.3，1.2】左右 请问 实在最后输出是使用sigmoid激活函数将值域限制在[0,1] ，还是模型输出后用torch.clamp 将值域裁剪呢 
期待你的解惑

**Comment by caiyuanhao1998** [comment]
你好，感谢关注，如果需要将值域映射在 [0,1]，建议直接 min-max normalization
